test/danetest.c

```diff
@@ -133,43 +115,38 @@ static STACK_OF(X509) *load_chain(BIO *fp, int nelem)
         if (strcmp(name, PEM_STRING_X509) == 0
             || strcmp(name, PEM_STRING_X509_TRUSTED) == 0
             || strcmp(name, PEM_STRING_X509_OLD) == 0) {
-            d2i_X509_t d = strcmp(name, PEM_STRING_X509_TRUSTED) ?
-                d2i_X509_AUX : d2i_X509;
-            X509 *cert = d(0, &p, len);
-
-            if (cert == 0 || (p - data) != len)
-                errtype = "certificate";
-            else if (sk_X509_push(chain, cert) == 0) {
-                perror("malloc");
+            d2i_X509_t d = strcmp(name, PEM_STRING_X509_TRUSTED)
+                            ? d2i_X509_AUX : d2i_X509;
```

Comment:
The `if` is a special case, not a rule.  In this case, I see no controversy or readability issue with applying our 4 space indent.  IF you want to make align-with-assign-op the rule, then we need to change our coding style, `util/openssl-format-source` and a whoooole lot of source.  Are you ready to make that jump?